docs/src/solvers/stochastic_gradient_descent.md

```diff
@@ -0,0 +1,24 @@
+# [Gradient Descent](@id StochasticGradientDescentSolver)
+
+```@meta
+CurrentModule = Manopt
+```
+
+```@docs
+  stochastic_gradient_descent
```

Comment:
The space shouldn't be needed before the function name?
```suggestion
stochastic_gradient_descent
```

Reply:
Thanks for noticing.

Reply:
no problem, I was browsing the PR for my own instruction :)